From 617d1f34be050a7b8a324bb8854a44c85f965909 Mon Sep 17 00:00:00 2001
From: Christophe Marin <christophe@krop.fr>
Date: Thu, 26 Oct 2023 21:03:47 +0200
Subject: [PATCH] Disable some plugins

- The updateinfo plugin only works when using the installer

- The clangformat plugin requires a clang patch which was never accepted upstream [1]

[1] https://reviews.llvm.org/D53072
---
 src/plugins/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/plugins/CMakeLists.txt b/src/plugins/CMakeLists.txt
index 2ec7c5a..f11b8e2 100644
--- a/src/plugins/CMakeLists.txt
+++ b/src/plugins/CMakeLists.txt
@@ -9,7 +9,7 @@ add_subdirectory(extensionmanager)
 add_subdirectory(helloworld)
 add_subdirectory(imageviewer)
 add_subdirectory(screenrecorder)
-add_subdirectory(updateinfo)
+# add_subdirectory(updateinfo)
 add_subdirectory(welcome)
 
 # Level 2: (only depends on Level 1 and below)
@@ -76,7 +76,7 @@ if (WITH_QMLDESIGNER)
   add_subdirectory(qmlprojectmanager)
 endif()
 add_subdirectory(python)
-add_subdirectory(clangformat)
+# add_subdirectory(clangformat)
 add_subdirectory(cmakeprojectmanager)
 
 # Level 7:
-- 
2.48.1

